cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Validate.hs

```diff
@@ -45,15 +45,15 @@ validateEpochRewards tracer earnedEpochNo rmap = do
     if actual /= expected
         then do
           liftIO . logWarning tracer $ mconcat
-                      [ "validateEpochRewards: rewards spendable in epoch "
+                      [ "Validate Epoch Rewards: rewards spendable in epoch "
```

Comment:
That is not actually an improvement. `validateEpochRewards` is the function name.